### PR TITLE
set no activation context on provisional tab when navigate to

### PR DESCRIPTION
**Pull request by heejaechang**
when PreferProvisionalTab option is added, work is done only in OpenDocument, but not in NavigateTo making the option useless in some cases since NavigateTo makes focus to move to provisional tab right after OpenDocument call which make sure provisional tab to not activated.

fix is following editor team's recommendation from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/402396

### CODE REVIEW

**Comment by heejaechang** [comment]
tagging @jinujoseph @dotnet/roslyn-ide @olegtk

File: src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs [hunk 1]
```diff
@@ -317,9 +317,34 @@ private bool NavigateTo(ITextBuffer textBuffer, VsTextSpan vsTextSpan)
                     return false;
                 }
 
-                return ErrorHandler.Succeeded(
-                    textManager.NavigateToLineAndColumn2(
-                        vsTextBuffer, VSConstants.LOGVIEWID.TextView_guid, vsTextSpan.iStartLine, vsTextSpan.iStartIndex, vsTextSpan.iEndLine, vsTextSpan.iEndIndex, (uint)_VIEWFRAMETYPE.vftCodeWindow));
+                if (options.GetOption(NavigationOptions.PreferProvisionalTab))
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs]: So this is copy/paste from the OpenDocument call earlier? Should we just be surrounding the calls to OpenDocument and NavigateTo with a single NewDocumentStateScope?

Comment (heejaechang) [reply src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs]: oh ya. I somehow thought OpenDocument has public. but it is private. so. will do.

**Comment by heejaechang** [comment]
@jinujoseph are we taking this to 16? or should I retarget it to master?

**Comment by jinujoseph** [comment]
lets retarget to 16.1.preview 1

**Pull request merged**